README.md

```diff
@@ -1 +1,10 @@
-# Hello
\ No newline at end of file
+Project Name: 
+
+Hello Microverse
+
+Description: 
+
+This project is an exercise for Microverse students to get familiar with git, github, 
+
+github flow and linters for HTML and CSS.
+
```

Comment:
- [ ] As requested by the previous reviewer, kindly update your README file according to [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) (copy the sections and fill them out with the information of your project) this way your project documentation will be professional ✨